pytorch_lightning/accelerators/accelerator.py

```diff
@@ -79,15 +79,14 @@ def setup_environment(self) -> None:
         """
         self.training_type_plugin.setup_environment()
 
-    def setup(self, trainer: 'pl.Trainer', model: 'pl.LightningModule') -> None:
+    def setup(self, trainer: 'pl.Trainer') -> None:
         """
         Setup plugins for the trainer fit and creates optimizers.
 
         Args:
             trainer: the trainer instance
-            model: the LightningModule
         """
-        self.setup_training_type_plugin(model)
+        self.setup_training_type_plugin()
```

Comment:
small note: This could be an issue if we ever decide to expose the Accelerator API.

Reply:
Why an issue? There's the `connect` hook to connect the model already.

`self.model` should be available for the plugin when `setup` is called

Reply:
@ananthsub We could